cmd/earthly/main.go

```diff
@@ -2575,18 +2575,21 @@ func (app *earthlyApp) actionConfig(c *cli.Context) error {
 
 	var outConfig []byte
 
-	switch {
-	case args[1] == "--delete":
-		outConfig, err = config.Delete(inConfig, args[0])
-		if err != nil {
-			return errors.Wrap(err, "delete config")
-		}
-	case args[1] == "--help" || args[1] == "-h":
+	switch args[1] {
+	case "-h":
+		fallthrough
+	case "--help":
```

Comment:
you can combine these into a single line:
```suggestion
	case "--help", "-h":
```

Reply:
💯